nrf_security/info.txt

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
What does this file do?

Reply:
looks like an MBEDTLS config file

Reply:
this file should have been removed.
Used internally during development.
Will remove.